### PR TITLE
Add introduction to fork on README

### PR DESCRIPTION
**Pull request by connortann**
Following the discussion https://github.com/dsgibbons/shap/discussions/1#discussioncomment-5789209 , I've taken a stab an some introductory text on the README, to help draw attention to the purpose of the fork.

I thought at this stage, it probably makes sense to put this information somewhere prominent, as a new CONTRIBUTING.md might not be noticed by people casually browsing the repo. For a title, how about "SHAP Community Fork"?

I haven't written a CONTRIBUTING.md in this PR, so that could be useful to have to flesh out the details of the how the pool of maintainers is managed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
+# SHAP Community Fork
 
+This repository is a fork of Scott-Lundberg's popular [shap](https://github.com/slundberg/shap) library. Unfortunately, the original shap repo is not currently maintained, so this fork attempts to fix shap's current issues and merge old PRs. See the general discussion on the purpose of this repo [here](https://github.com/dsgibbons/shap/discussions/11).
```
Comment (dsgibbons) [inline README.md:3]: Scott-Lundberg -> Scott Lundberg

File: README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+
+**New contributors are very welcome**, so please feel free to get involved!
+
+We are eager to build a broad pool of maintainers, to avoid the mistake of having a single person responsible for the entire repostitory. This repo adopts a _liberal contribution governance model_, where project decisions are based on a consensus seeking process. For more information, see [here](https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951).
```
Comment (dsgibbons) [inline README.md:7]: to avoid ~~the mistake of~~ having a single person

File: README.md (hunk 1 above)
Comment (dsgibbons) [inline README.md:3]: > Unfortunately, the original shap repo is not currently maintained, so this fork attempts to fix shap's current issues and merge old PRs

 Unfortunately, the original shap repo is not currently maintained. This fork attempts to fix shap's current issues and merge old PRs.

**Review by dsgibbons — CHANGES_REQUESTED** [review]
Looks good. I've made a few minor suggestions

**Comment by connortann** [comment]
Thanks for the review comments! Feel free to change the title if you have something else in mind other than `Shap Community Fork`

**Review by thatlittleboy — APPROVED** [review]
cat-wearing-shades-letsgo

**Review by dsgibbons — APPROVED** [review]
Thank you @connortann - I like `SHAP Community Fork` - sounds wholesome

**Pull request merged**